docs/development/extensions-contrib/kafka-emitter.md

```diff
@@ -36,16 +36,17 @@ to monitor the status of your Druid cluster with this extension.
-| `druid.emitter.kafka.clusterName`                  | Optional value to specify the name of your Druid cluster. It can help make groups in your monitoring environment.                         | no        | none                  |
+| Property                                    | Description                                                                                                                                                | Required | Default               |
+|---------------------------------------------|------------------------------------------------------------------------------------------------------------------------------------------------------------|-----------|-----------------------|
+| `druid.emitter.kafka.bootstrap.servers`     | Comma-separated Kafka broker. (`[hostname:port],[hostname:port]...`)                                                                                       | yes       | none                  |
+| `druid.emitter.kafka.event.types`           | Comma-separated event types. <br/>Supported types are `alerts`, `metrics`, `requests`, and `segment_metadata`.                                             | no        | `["metrics", "alerts"]` |
+| `druid.emitter.kafka.metric.topic`          | Kafka topic name for emitter's target to emit service metrics. If `event.types` contains `metrics`, this field cannot be empty.                            | no        | none                  |
+| `druid.emitter.kafka.alert.topic`           | Kafka topic name for emitter's target to emit alerts. If `event.types` contains `alerts`, this field cannot empty.                                         | no        | none                  |
+| `druid.emitter.kafka.request.topic`         | Kafka topic name for emitter's target to emit request logs. If `event.types` contains `requests`, this field cannot be empty.                              | no        | none                  |
+| `druid.emitter.kafka.segmentMetadata.topic` | Kafka topic name for emitter's target to emit segment metadata. If `event.types` contains `segment_metadata`, this field cannot be empty.                  | no        | none                  |
+| `druid.emitter.kafka.producer.config`       | JSON configuration to set additional properties to Kafka producer.                                                                                         | no        | none                  |
+| `druid.emitter.kafka.clusterName`           | Optional value to specify the name of your Druid cluster. It can help make groups in your monitoring environment.                                          | no        | none                  |
+| `druid.emitter.kafka.extra.dimensions`      | Optional JSON configuration to specify a map of string extra dimensions for the events emitted. These can help make groups in your monitoring environment. | no        | none                  |
```

Comment:
The [Prometheus](https://druid.apache.org/docs/latest/development/extensions-contrib/prometheus/#configuration) emitter has a similar config `druid.emitter.prometheus.extraLabels`, so for consistency, call this `extraLabels` as well instead of `extra.dimensions`?



<br class="Apple-interchange-newline">

Reply:
labels is a prometheus construct i chose dimensions because that is more of a druid construct.

Reply:
[dogstatsd](https://github.com/apache/druid/blob/e184d24a74dbe425e2f80facbe09071049dcfb35/extensions-contrib/statsd-emitter/src/main/java/org/apache/druid/emitter/statsd/StatsDEmitterConfig.java#L70) has something similar as well named constant tags... granted its not a map, its just a list of tags

Reply:
Works for me. `extraDimensions`